spec/lib/tasks/update_database_records_in_big_query_rake_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   it 'queues the export vacancy records to big query job' do
-    expect { task.execute }.to have_enqueued_job(ExportVacancyRecordsToBigQueryJob)
+    # expect { task.execute }.to have_enqueued_job(ExportVacancyRecordsToBigQueryJob)
```

Comment:
Can you please remove these rather than commenting them out? They'll still be in the history if we need them.